Core/CoreUITests/Assignments/AssignmentsTests.swift

```diff
@@ -17,9 +17,11 @@
 //
 
 import XCTest
-import TestsFoundation
+@testable import TestsFoundation
+
+class AssignmentsTests: CoreUITests {
+    override var abstractTestClass: CoreUITests.Type { return AssignmentsTests.self }
```

Comment:
This test is specific to the Student app, and should probably be moved to StudentUITests.